### ISSUE TITLE
Property:thematicFocus

### ISSUE THREAD

**Issue by mtrekels** [tdwg/cd#62]
|  |  |
| ---- | ---- |
| **Definition** | A texual description of the collection and the characteristics and its idiosyncrasies |
| **Dimension** |  |
| **Existing property** |  |
| **Existing class** |  |
| **Existing property identifier** |  |
| **Format** | Text |
| **Required** |  |
| **Repeatable** |  |
| **Constraints** |  |
| **Examples** |  |
| **Notes** |  |

**Comment by mswoodburn** [comment]
Suggestion from a comment on the index sheet to use dcterms:abstract (https://www.dublincore.org/specifications/dublin-core/collection-description/collection-application-profile/#coldctermsabstract)

**Comment by debpaul** [comment]
**Example**: (high level; from NHM) The Museum’s Lepidoptera collection contains approximately 8,712,000 specimens in 80,000 drawers. The Lepidoptera specimens are divided into two separate collections, British and international, both of which are rich in type specimens.

Would this be a suitable, high-level example?

**Comment by debpaul** [comment]
**Example**: (high level; from NHM) The Museum’s orthopteroid collection contains an estimated 785,824 specimens, including 5,567 primary and 15,051 secondary type specimens. 20,618 are Type specimens.

**Comment by qgroom** [comment]
I would not put actual numbers in an example, because they grow old quickly and we don't want people to put numbers here,

**Comment by dbloom** [comment]
I agree with @qgroom completely.  This is probably a good rule of thumb when thinking about content in any cd field.

**Comment by mswoodburn** [comment]
Also agree re the numbers.

Suggestion for updated definition: "A textual description of the collection's main foci (e.g. geographic, taxonomic or historical), strengths and idiosyncrasies."

Some potential examples: 
"Worldwide, with specific strength in Madagascar, New Caledonia, Brazil"
"Belgian and DRC prehistoric sites (middle paleolithic to protohistory), Belgian historical human remains. World reference collection."
"Primarily marine Crustacea, with worldwide representation but with historical strengths in eastern Pacific Decapoda. Major historical component is from the Allan Hancock Expeditions along the west coast of the Americas in the early 20th century."
"Includes a unique collection of ferromanganese crusts and nodules from shelf sea areas; mainly from the Baltic and Barents Sea."

**Comment by mswoodburn** [comment]
Maybe also add a note that numeric values should be handled by the metric classes wherever possible.
